### PR TITLE
Spelling suggestions + corrected link in README

### PR DESCRIPTION
**Pull request by WKobes**
Not part of the working group and quite new to IETF contributions, but these stood out while reading the drafts. Let me know if this is not the right way to contribute.

Also corrected the URL to IETF in the README.md, since the previous one did not work.

### CODE REVIEW

File: draft-schwartz-httpbis-dns-alt-svc.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ records, the server operator MUST set the overall TTL to the minimum
 of the "max age" values (following Section 5.2 of {{!RFC2181}}).
 Each RR can contain multiple alt-values separated by commas, as described
 in Section 3 of {{!AltSvc}}.  As specified there, "the order of values
-relects the server's preference".
+reflects the server's preference".
```
Comment (bemasc) [inline draft-schwartz-httpbis-dns-alt-svc.md:183]: This draft is obsolete, so I don't think we should make any further changes to it.

File: draft-ietf-dnsop-svcb-https.md [hunk 2]
```diff
@@ -1711,7 +1711,7 @@ Global Scoped Entry Registry:
 
 # Acknowledgments and Related Proposals
 
-There have been a wide range of proposed solutions over the years to
+There has been a wide range of proposed solutions over the years to
```
Comment (bemasc) [inline draft-ietf-dnsop-svcb-https.md:1714]: This seems like at least a debatable point of grammar.

Comment (MikeBishop) [reply draft-ietf-dnsop-svcb-https.md:1714]: Agreed.  I think either would be fine, and I'm inclined to let the RFC Editor sort it out at this point.

**Review by MikeBishop — COMMENTED** [review]
At this point of the process, I tend to think we just see if the RFC Editor misses the space.

File: draft-ietf-dnsop-svcb-https.md [hunk 3]
```diff
@@ -309,7 +309,7 @@ Clients MUST consider an RR malformed if:
 
 * the end of the RDATA occurs within a SvcParam.
 * SvcParamKeys are not in strictly increasing numeric order.
-* the SvcParamValue for an SvcParamKey does not have the expected format.
+* the SvcParamValue for a SvcParamKey does not have the expected format.
```
[the remark this answers is not in the document]
Comment (WKobes) [reply draft-ietf-dnsop-svcb-https.md:312]: Agreed, although the rest of the document (e.g. two lines above it) tends to use 'a'

**Comment by MikeBishop** [comment]
Closing per discussion and lack of AD approval.

**Pull request closed without merging**